drivers/espi/espi_it8xxx2.c

```diff
@@ -687,22 +696,130 @@ static int espi_it8xxx2_write_lpc_request(const struct device *dev,
 }
 
 #ifdef CONFIG_ESPI_OOB_CHANNEL
+/* eSPI cycle type field */
+#define ESPI_OOB_CYCLE_TYPE          0x21
+#define ESPI_OOB_TAG                 0x00
+#define ESPI_OOB_TIMEOUT_MS          200
+
+struct espi_oob_msg_packet {
+	uint8_t cycle_type;
+	uint8_t tag_len_bit_11_8;
+	uint8_t len_bit_7_0;
+	uint8_t data_byte[ESPI_IT8XXX2_OOB_MAX_PAYLOAD_SIZE];
```

Comment:
You don't need to allocate the payload here.  Use `uint8_t data_byte[0];` to create the data_byte field at the correct offset, but without allocating data.

Reply:
Done.